### PR TITLE
[FIX][11.0] website_form_builder travis warning

### PR DESCRIPTION
**Pull request by tbaden**
issue https://github.com/OCA/website/issues/603 : 
fix of:
<details><summary>logs</summary>
```

************* Module website_form_builder
website_form_builder/static/src/js/tour.js:11: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
website_form_builder/static/src/js/tour.js:15: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
website_form_builder/static/src/js/tour.js:20: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:21: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:22: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:23: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:24: [W7903(javascript-lint), ] Expected indentation of 8 spaces but found 4. [Error/indent]
website_form_builder/static/src/js/tour.js:25: [W7903(javascript-lint), ] Expected indentation of 8 spaces but found 4. [Error/indent]
website_form_builder/static/src/js/tour.js:26: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:27: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:28: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:29: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:30: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:31: [W7903(javascript-lint), ] Line 31 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/tour.js:31: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:32: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:33: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:34: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:35: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:36: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:37: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:38: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:39: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:40: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:41: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:42: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:43: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:44: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:45: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:46: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:47: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:48: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:49: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:50: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:51: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:52: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:53: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:54: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:55: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:56: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:57: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:58: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:59: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:60: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:61: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:62: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:63: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:64: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:65: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:66: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:67: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:68: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:69: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:70: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:71: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:72: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:73: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:74: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:75: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:76: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:77: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:78: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:79: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:80: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:81: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:82: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:83: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:84: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:85: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:86: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:87: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:88: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:89: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:90: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:91: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:92: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:93: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:94: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:95: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:96: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:97: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:98: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:99: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js💯 [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:101: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:102: [W7903(javascript-lint), ] Line 102 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/tour.js:102: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:103: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:104: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:105: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:105: [W7903(javascript-lint), ] Line 105 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/tour.js:106: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:107: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:108: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:109: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:110: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:111: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:112: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:113: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:114: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:115: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:116: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:117: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:118: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:119: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:120: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:121: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:122: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:123: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:124: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:125: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:126: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:127: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:128: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:129: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:130: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:131: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:132: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:133: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:134: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:135: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:136: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:137: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:138: [W7903(javascript-lint), ] Line 138 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/tour.js:138: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:139: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:140: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:141: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:142: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:143: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:144: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:145: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:146: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:147: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:148: [W7903(javascript-lint), ] Line 148 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/tour.js:148: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:149: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:150: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:151: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:152: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:153: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:154: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 12. [Error/indent]
website_form_builder/static/src/js/tour.js:155: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 8. [Error/indent]
website_form_builder/static/src/js/tour.js:156: [W7903(javascript-lint), ] Expected indentation of 8 spaces but found 4. [Error/indent]
website_form_builder/static/src/js/snippets.js:14: [W7903(javascript-lint), ] 'ServicesMixin' is assigned a value but never used. [Error/no-unused-vars]
website_form_builder/static/src/js/snippets.js:26: [W7903(javascript-lint), ] Missing JSDoc for parameter 'servicesMixin'. [Error/valid-jsdoc]
website_form_builder/static/src/js/snippets.js:31: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
website_form_builder/static/src/js/snippets.js:34: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 20. [Error/indent]
website_form_builder/static/src/js/snippets.js:35: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 20. [Error/indent]
website_form_builder/static/src/js/snippets.js:36: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 20. [Error/indent]
website_form_builder/static/src/js/snippets.js:36: [W7903(javascript-lint), ] Unexpected line break after this opening brace. [Error/object-curly-newline]
website_form_builder/static/src/js/snippets.js:37: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:38: [W7903(javascript-lint), ] Expected indentation of 24 spaces but found 28. [Error/indent]
website_form_builder/static/src/js/snippets.js:39: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:40: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:41: [W7903(javascript-lint), ] Expected indentation of 24 spaces but found 28. [Error/indent]
website_form_builder/static/src/js/snippets.js:42: [W7903(javascript-lint), ] Expected indentation of 24 spaces but found 28. [Error/indent]
website_form_builder/static/src/js/snippets.js:43: [W7903(javascript-lint), ] Expected indentation of 24 spaces but found 28. [Error/indent]
website_form_builder/static/src/js/snippets.js:44: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:45: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:46: [W7903(javascript-lint), ] Expected indentation of 20 spaces but found 24. [Error/indent]
website_form_builder/static/src/js/snippets.js:46: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
website_form_builder/static/src/js/snippets.js:47: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 16. [Error/indent]
website_form_builder/static/src/js/snippets.js:48: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 20. [Error/indent]
website_form_builder/static/src/js/snippets.js:49: [W7903(javascript-lint), ] Expected indentation of 12 spaces but found 16. [Error/indent]
website_form_builder/static/src/js/snippets.js:55: [W7903(javascript-lint), ] Missing JSDoc for parameter 'servicesMixin'. [Error/valid-jsdoc]
website_form_builder/static/src/js/snippets.js:61: [W7903(javascript-lint), ] Missing space before function parentheses. [Error/space-before-function-paren]
website_form_builder/static/src/js/snippets.js:71: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
website_form_builder/static/src/js/snippets.js:83: [W7903(javascript-lint), ] Expected line before comment. [Error/lines-around-comment]
website_form_builder/static/src/js/snippets.js:231: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
website_form_builder/static/src/js/snippets.js:313: [W7903(javascript-lint), ] Line 313 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/snippets.js:445: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
website_form_builder/static/src/js/snippets.js:467: [W7903(javascript-lint), ] Expected indentation of 16 spaces but found 20. [Error/indent]
website_form_builder/static/src/js/snippets.js:508: [W7903(javascript-lint), ] Line 508 exceeds the maximum line length of 80. [Error/max-len]
website_form_builder/static/src/js/snippets.js:532: [W7903(javascript-lint), ] Missing trailing comma. [Error/comma-dangle]
website_form_builder/static/src/js/widgets.js:4: [W7903(javascript-lint), ] Expected indentation of 0 spaces but found 1. [Error/indent]
website_form_builder/static/src/js/widgets.js:5: [W7903(javascript-lint), ] Expected indentation of 4 spaces but found 5. [Error/indent]
```
</details>

### CODE REVIEW

**Review by yajo — CHANGES_REQUESTED** [review]
Still some errors: https://travis-ci.org/OCA/website/jobs/530023787#L743-L754

**Comment by tbaden** [comment]
still WIP, will check this later today

**Comment by tbaden** [comment]
@Yajo are you using eslint by any chance?

**Comment by tbaden** [comment]
@Yajo is this the correct param type to use here?
https://github.com/OCA/website/blob/9345c961e6a98b7d121c3bd5a930d659760260e0/website_form_builder/static/src/js/snippets.js#L28

File: website_form_builder/static/src/js/snippets.js [hunk 1]
```diff
@@ -222,13 +222,11 @@ odoo.define('website_form_builder.snippets', function (require) {
...
-                    method: 'formbuilder_whitelist',
+                    model: "ir.model.fields",
+                    method: "formbuilder_whitelist",
                     args: [this.controller_data().model_name, fields],
-                    kwargs: {
-                        context: weContext.get(),
-                    },
-                    async: false
+                    kwargs: {context: weContext.get()},
+                    async: false,
```
Comment (yajo) [inline website_form_builder/static/src/js/snippets.js]:
```suggestion
                    "async": false,
```

This is a reserved word in modern JS.

File: website_form_builder/static/src/js/snippets.js [hunk 2]
```diff
@@ -310,9 +308,10 @@ odoo.define('website_form_builder.snippets', function (require) {
                 // Nothing to reset here
                 return;
             }
-            return authorized_fields(this.controller_data().model_name, this).done(
-                $.proxy(this._ask_model_field, this)
-            );
+            return authorized_fields(
+                this.controller_data().model_name, this)
+                .done($.proxy(this._ask_model_field, this)
+                );
```
Comment (yajo) [inline website_form_builder/static/src/js/snippets.js]:
```suggestion
            );
```

**Review by yajo — CHANGES_REQUESTED** [review]
Please abstain from changing unnecessary things:

- Multiple variable declarations in one statement.
- Single vs double quotes.
- Multiline dicts with a single key.

Just remove the warnings, but don't apply changes that are not required for that please. The git history is helpful to review why things changed, so reducing the diff is helpful for maintenance.

**Comment by tbaden** [comment]
done

**Review by yajo — APPROVED** [review]

File: website_form_builder/static/src/js/snippets.js [hunk 3]
```diff
@@ -26,27 +25,27 @@ odoo.define('website_form_builder.snippets', function (require) {
     /**
      * Lazily ask just once for models.
      *
+     * @param {Class} servicesMixin
```
Comment (yajo) [inline website_form_builder/static/src/js/snippets.js]: In JS there are no classes; they are `Object`s.

```suggestion
     * @param {Object} servicesMixin
```

File: website_form_builder/static/src/js/snippets.js [hunk 4]
```diff
@@ -56,9 +55,10 @@ odoo.define('website_form_builder.snippets', function (require) {
      * Lazily load just once authorized fields for given model.
      *
      * @param {String} model Model technical name.
+     * @param {Class} servicesMixin
```
Comment (yajo) [inline website_form_builder/static/src/js/snippets.js]:
```suggestion
     * @param {Object} servicesMixin
```

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by tbaden** [comment]
done

**Review by yajo — APPROVED** [review]

**Pull request merged**